CHANGES.rst

```diff
@@ -91,6 +91,8 @@ New Features
   - Write comment lines in Table metadata during output for all basic formats,
     IPAC, and fast writers. This functionality can be disabled with
     ``comment=False``. [#3255]
```

Comment:
An annoying nitpick--now there should be a blankline here (I was worried that the merge=union wouldn't always handle this properly, but it's still better than not having it at all.  Now if only it were possible to run custom merge strategies on GH....)